### PR TITLE
mvn verify runs successfully

### PR DESCRIPTION
**Pull request by vrajat**
Previously `mvn integration-tests` was used to run integration tests. However that command does not error out if integration tests fail. The right command to use is `mvn verify`. To make `mvn verify`, the following changes were required:
* Restrict header checks to io.debezium classes
* Remove usage of log4j
* Remove usage of star imports

### CODE REVIEW

File: cdcsdk-server/cdcsdk-server-s3/src/main/java/com/yugabyte/cdcsdk/sink/s3/config/S3SinkConnectorConfig.java [hunk 1]
```diff
@@ -921,7 +921,9 @@ public String toString() {
     }
 
     public static void main(String[] args) {
+        // CHECKSTYLE:OFF
         System.out.println(getConfig().toEnrichedRst());
+        // CHECKSTYLE:ON
```
Comment (vaibhav-yb) [inline cdcsdk-server/cdcsdk-server-s3/src/main/java/com/yugabyte/cdcsdk/sink/s3/config/S3SinkConnectorConfig.java:926]: Do these comments indicate something?

File: pom.xml [hunk 2]
```diff
@@ -427,4 +426,4 @@
             </modules>
         </profile>
     </profiles>
-</project>
\ No newline at end of file
+</project>
```
Comment (vaibhav-yb) [inline pom.xml:429]: Newline missing at the end of the file.

File: support/checkstyle/src/main/resources/checkstyle.xml [hunk 3]
```diff
@@ -173,4 +173,4 @@
         <property name="file" value="${checkstyle.suppressions.file}"/>
     </module>
 
-</module>
\ No newline at end of file
+</module>
```
Comment (vaibhav-yb) [inline support/checkstyle/src/main/resources/checkstyle.xml:176]: Newline missing here at the end of the file.

**Review by vaibhav-yb — APPROVED** [review]
Approving preemptively. Also added minor comments related to adding a newline.

File: cdcsdk-server/cdcsdk-server-s3/src/main/java/com/yugabyte/cdcsdk/sink/s3/config/S3SinkConnectorConfig.java (hunk 1 above)
Comment (vrajat) [reply cdcsdk-server/cdcsdk-server-s3/src/main/java/com/yugabyte/cdcsdk/sink/s3/config/S3SinkConnectorConfig.java:926]: Yes. Checkstyle is turned off for this line and then turned on again

**Review by Sumukh-Phalgaonkar — APPROVED** [review]

**Pull request merged**